### ISSUE TITLE
Chargement d'un dossier

### ISSUE THREAD

**Issue by michelwend** [broncowdd/BoZoN#73]
Peut-être qu'en plus du multi utilisateur pour la version 2 le chargement d'un dossier et de tout ce qu'il contient (sous-dossiers et fichiers) sera possible? Même si c'est http://www.cyrille-borne.com/article2190/uploader-des-dossiers-dans-bozon nous dit que cela est impossible. J'ai souvenir avoir demander le multi utilisateur il y a longtemps et cela semblait également impossible mais aujourd'hui on y est et c'est super! Merci encore pour le boulot accompli.

**Comment by broncowdd** [comment]
Salut et de rien ;-)
Le problème vient de la soumission d'un formulaire et de javascript, pas d'une limite de bozon, malheureusement...

**Comment by michelwend** [comment]
Et c'est impossible à contourner, j'imagine?

**Comment by cborne** [comment]
le coup du zip est une façon de le contourner. Tu peux zipper un dossier 
avec une grosse arborescence et dézipper dedans, ça fait un clic de plus

**Comment by broncowdd** [comment]
Je ne m'avoue pas vaincu: si c'est possible de détecter un drop de dossier et de le parcourir, l'upload ne serait pas le plus dur...

**Comment by cborne** [comment]
UN NOUVEAU DEFI POUR BRONCO, DE NOUVEAUX BUGS POUR BOZON.

Cher public ne démarquez pas, la partie n'est pas terminée.

**Comment by broncowdd** [comment]
J'ai trouvé une lib js qui semble faire le job (à tester sur autre chose que chromium quand même)...
je regarderai à tête reposée, parce que 100ko juste pour l'upload de fichiers... :-/
http://protonet.github.io/plupload/examples/drag_and_drop.html

**Comment by broncowdd** [comment]
un nouvel espoir ?

**Comment by Pafzedog** [comment]
apparemment ça ne fonctionne qu'avec chrome (webkit) non ? comme tous les uploader js que j'ai vu d'ailleurs.
Il semblerait que pour l'instant ça soit pas possible avec des navigateurs basés sur d'autres moteurs.

J'ai trouvé ça aussi: http://fineuploader.com/demos.html juste du js, je crois que le script de base fait 64 ko.
ça permet au moins de glisser plusieurs fichiers d'un coup, c'est déjà ça en attendant que FF permette le truc :/

**Comment by broncowdd** [comment]
Oui, ça ne marche qu'avec webkit, c'est le souci... Même si firefox adopte webkit, ça pose le problème pour les autres... 

Ceci dit, on peut déjà envoyer plusieurs fichiers en même temps ;-)

Le 31 janvier 2016 23:32:23 CET, Pafzedog <notifications@github.com> a écrit :
>apparemment ça ne fonctionne qu'avec chrome (webkit) non ? comme tous
>les uploader js que j'ai vu d'ailleurs.
>Il semblerait que pour l'instant ça soit pas possible avec des
>navigateurs basés sur d'autres moteurs.
>
>J'ai trouvé ça aussi: http://fineuploader.com/demos.html juste du js,
>je crois que le script de base fait 64 ko.
>ça permet au moins de glisser plusieurs fichiers d'un coup, c'est déjà
>ça en attendant que FF permette le truc :/
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/broncowdd/BoZoN/issues/73#issuecomment-177641798

-- 
Envoyé de mon appareil Android avec K-9 Mail. Veuillez excuser ma brièveté.

**Comment by michelwend** [comment]
Bon le défi est lancé, patience, patience...

**Comment by Pafzedog** [comment]
> Ceci dit, on peut déjà envoyer plusieurs fichiers en même temps ;-)
oui mais pas avec à partir de la sélection quand on clique sur la zone d'upload (je chipote) .
En fait je n'utilise pratiquement jamais le drag'n drop et du coup je viens de me rendre compte qu'on pouvait envoyer plusieurs fichiers ;)

D'ailleurs il faudrait faire un sondage, est-ce que le drag'n drop est vraiment utilisé par les gens(en général) ? J'ai jamais trouvé ça pratique en fait, mes fenêtres (navigateur ou gestionnaire de fichier) sont toujours maximisées et du coup c'est pas vraiment plus simple que de cliquer et sélectionner. Mais bon, c'est un autre débat ;)

**Comment by broncowdd** [comment]
Pour moi, rien ne vaut le drag & drop ^^
